host-configs/TOTAL/pecan-GPU.cmake

```
@@ -3,14 +3,13 @@ include(${CMAKE_CURRENT_LIST_DIR}/pecan-CPU.cmake)
 
 # Now let's add what's dedicated to GPU.
 set(ENABLE_CUDA ON CACHE PATH "" FORCE)
-set(CUDA_TOOLKIT_ROOT_DIR /hrtc/apps/cuda/10.2.89/x86_64 CACHE PATH "")
+set(CUDA_TOOLKIT_ROOT_DIR /hrtc/apps/cuda/11.5.119/x86_64 CACHE PATH "")
```

maybe this is actually:
```suggestion
set(CUDA_TOOLKIT_ROOT_DIR /hrtc/apps/cuda/11.5.119/x86_64/centos7 CACHE PATH "")
```